common-lib/src/main/scala/com/gu/mediaservice/model/UsageRights.scala

```diff
@@ -60,24 +60,24 @@ object UsageRights {
       val supplier = (json \ "supplier").asOpt[String]
 
       (category flatMap {
-        case "chargeable"                => json.asOpt[Chargeable]
-        case "agency"                    => json.asOpt[Agency]
-        case "commissioned-agency"       => json.asOpt[CommissionedAgency]
-        case "PR Image"                  => json.asOpt[PrImage]
-        case "handout"                   => json.asOpt[Handout]
-        case "screengrab"                => json.asOpt[Screengrab]
-        case "guardian-witness"          => json.asOpt[GuardianWitness]
-        case "social-media"              => json.asOpt[SocialMedia]
-        case "obituary"                  => json.asOpt[Obituary]
-        case "staff-photographer"        => json.asOpt[StaffPhotographer]
-        case "contract-photographer"     => json.asOpt[ContractPhotographer]
-        case "commissioned-photographer" => json.asOpt[CommissionedPhotographer]
-        case "pool"                      => json.asOpt[Pool]
-        case "crown-copyright"           => json.asOpt[CrownCopyright]
-        case "contract-illustrator"      => json.asOpt[ContractIllustrator]
-        case "commissioned-illustrator"  => json.asOpt[CommissionedIllustrator]
-        case "creative-commons"          => json.asOpt[CreativeCommons]
-        case _                           => None
+        case "chargeable"                       => json.asOpt[Chargeable]
+        case "agency"                           => json.asOpt[Agency]
+        case "commissioned-agency"              => json.asOpt[CommissionedAgency]
+        case "PR Image"                         => json.asOpt[PrImage]
+        case "handout"                          => json.asOpt[Handout]
+        case "screengrab"                       => json.asOpt[Screengrab]
+        case "guardian-witness"                 => json.asOpt[GuardianWitness]
+        case "social-media"                     => json.asOpt[SocialMedia]
+        case "obituary"                         => json.asOpt[Obituary]
```

Comment:
Yep...though now theres loads of whitespace changes in the diff :crying_cat_face: